UnicornRender/utility/sources/Math.cpp

```diff
@@ -77,6 +77,29 @@ glm::vec3 RotateAroundPoint(glm::vec3 originalTranslation, float radians, glm::v
     return outputTranslation;
 }
 
+glm::mat4 AssimpMatrixToGlm(aiMatrix4x4 const& from)
```

Comment:
I think `glm::make_mat4` will do the job better.

Reply:
@Warezovvv what about this one?

Reply:
I think it will not. 
For future PR comments with some code improvements I propose also write why you think or why should I do as you've wrote above. Also I've tried with glm::make_mat4. Its building matrix but wrong one. Im not interested to fix that.

Reply:
If does build the wrong matrix, it's only because you are not using the same major for storing the matrix. And I strongly recommend, you do use the same major for every matrix in your code. In this case all you need to do is to transpose the matrix after the make call.